tools/integration_tests/util/operations/dir_operations.go

```diff
@@ -158,3 +159,23 @@ func DirSizeMiB(dirPath string) (dirSizeMB int64, err error) {
 
 	return dirSizeMB, err
 }
+
+func DeleteManagedFoldersInBucket(managedFolderPath, bucket string, t *testing.T) {
+	gcloudDeleteManagedFolderCmd := fmt.Sprintf("alpha storage rm -r gs://%s/%s", bucket, managedFolderPath)
+
+	_, err := ExecuteGcloudCommandf(gcloudDeleteManagedFolderCmd)
+	if err != nil && !strings.Contains(err.Error(), "The following URLs matched no objects or files") {
+		t.Fatalf(fmt.Sprintf("Error while deleting managed folder: %v", err))
+	}
+}
+
+func CreateManagedFoldersInBucket(managedFolderPath, bucket string, t *testing.T) {
+	// Delete if already exist.
+	DeleteManagedFoldersInBucket(managedFolderPath, bucket, t)
+	gcloudCreateManagedFolderCmd := fmt.Sprintf("alpha storage managed-folders create gs://%s/%s", bucket, managedFolderPath)
+
+	_, err := ExecuteGcloudCommandf(gcloudCreateManagedFolderCmd)
+	if err != nil {
+		t.Fatalf(fmt.Sprintf("Error while creating managed folder: %v", err))
```

Comment:
Thumb rule: use %w while formatting error.
Ref: https://stackoverflow.com/questions/61283248/format-errors-in-go-s-v-or-w

Reply:
Lint error - printf: fmt.Sprintf does not support error-wrapping directive %w (govet)